packages/react-search-ui/README.md

```diff
@@ -282,44 +478,321 @@ import { Sorting } from "@elastic/react-search-ui";
+| `clearFilters`      |                                                                                                                                                                            |        | Clear all filters                                                                           |
+| `setCurrent`        | Integer                                                                                                                                                                    |        |                                                                                             |
+| `setResultsPerPage` | Integer                                                                                                                                                                    |        |                                                                                             |
+| `setSearchTerm`     | String                                                                                                                                                                     |        |                                                                                             |
+| `setSort`           | `sortField` String - field to sort on<br/>`sortDirection` String - "asc" or "desc"                                                                                         |        |                                                                                             |
+| `trackClickThrough` | `documentId` String - The document ID associated with the result that was clicked<br/>`tag` - Array[String] Optional tags which can be used to categorize this click event |        | Report a click through event. A click through event is when a user clicks on a result link. |
+
+<a id="state"></a>
+
+#### State
+
+[back](#nav)
+
+State can be divided up into two types.
+
+1. Request State - Reflects the state of the most recent request. It is updated at the time a request is made.
+2. Result State - Result State is updated AFTER a response is received.
+
+For this reason, there are often two versions of state. For instance, `searchTerm` and `resultSearchTerm`. This can
+be relevant in the UI, where you might not want the search term on the page to change until AFTER a response is
+received, so you'd use the `resultSearchTerm` state.
+
+<a id="requeststate"></a>
+
+_Request State_
+
+| option           | type                     | required? | source                                                                        |
+| ---------------- | ------------------------ | --------- | ----------------------------------------------------------------------------- |
+| `current`        | Integer                  | optional  | current page number                                                           |
+| `filters`        | Array[Object]            | optional  | [Reference](https://swiftype.com/documentation/app-search/api/search/filters) |
```

Comment:
Perhaps [App Search Filters API Reference]() so it's explicit where they're going and what-for.